feature-group-definitions/promise-allsettled.yml

```
@@ -0,0 +1,17 @@
+name: Promise.allSettled()
+description: Promise.allSettled() static method
```

Descriptions are only for our own benefit as they aren't published yet, but I expect we will write a guideline and publish them later. I would argue that descriptions should say what a feature does, not just describe the API surface. Something like:

```suggestion
description: The Promise.allSettled() static method waits for an array of promises to settle (resolve or reject).
```

Just omitting descriptions in PRs is fine too.

---

This almost sounds like MDN short descriptions which we might be able to get programmatically. Would love to have a definition for the `description` field because your suggestion here would now be inconsistent with what you suggested in the Array PR...

---

The discussion in https://github.com/web-platform-dx/web-features/pull/598#discussion_r1486528424 is what we have so far.

I've tried to do "feature X does Y" in the active voice, sometimes with some extra commentary at the end.